### PR TITLE
updated to modify, add, delete to city

### PR DESCRIPTION
**Pull request by rrarima**
Hello, please review this code. Only changes made were to update current functions of state, and apply them to city.

### CODE REVIEW

File: api/v1/views/cities.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python3
+"""
+Creates a view for State objects that handles all default RESTFul API actions
+"""
+from models.state import State
+from models.state import City
```
Comment (jennyngu) [inline api/v1/views/cities.py]: Did you mean from models.city instead of models.state? The code ran without issues?

File: api/v1/views/cities.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+from models import storage
+from models.base_model import BaseModel
+from api.v1.views import app_views
+from flask import request, jsonify, abort
+
```
Comment (jennyngu) [inline api/v1/views/cities.py:11]: Ordering the imports based on classes and alphabetically would look a bit more cleaner

File: api/v1/views/cities.py (hunk 1 above)
Comment (rrarima) [reply api/v1/views/cities.py]: Oh yes it did run without any errors. Thanks for spotting/fixing that!

File: api/v1/views/cities.py (hunk 2 above)
Comment (rrarima) [reply api/v1/views/cities.py:11]: I appreciate that!

**Pull request merged**